content/pages/enterprise.html

```diff
@@ -0,0 +1,132 @@
+{# View this page at http://localhost:8080/pricing/enterprise/ #}
+
+<html>
+  <head>
+    <title>Read the Docs Enterprise Plans</title>
```

Comment:
This can be reduced, the title already has "Read the Docs" appended

```suggestion
    <title>Enterprise Plans</title>
```